test/zksync-tests/CheckOwnership.test.ts

```diff
@@ -0,0 +1,197 @@
+import { CommandType, RoutePlanner } from './shared/planner';
```

Comment:
Can you please move these tests to the `test/integration-tests`? I mean replace the Ethereum ones, instead of creating a new directory. I think it will be more convenient, at least for code review

Reply:
Yes